Tests/OTelConventionsTests/OTelAttributesTests.swift

```diff
@@ -0,0 +1,10 @@
+import OTelConventions
+import Testing
+
+struct OTelHttpAttributesTests {
```

Comment:
```suggestion
struct OTelHTTPAttributesTests {
```
Nitpick.

Reply:
Done in https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/bc3528abdcd6a1b158409a6a343041d71733fb84